### ISSUE TITLE
emscripten support

### ISSUE THREAD

**Issue by claudeha** [libpd/libpd#391]
Hi,

I have a fork of pure-data and libpd adding support for Emscripten.  I want to get them merged into pure-data and libpd upstream so my maintenance burden is reduced.  I suppose the libpd part is blocked on the pure-data part (which I'm hoping Miller might merge soon).  Filing this issue so I don't forget about the libpd part.  Comments welcome on the best way to update the code to current libpd and make a pull request when the time comes.

- https://github.com/claudeha/pure-data/tree/emscripten my fork of pure-data with changes for Emscripten
- https://github.com/pure-data/pure-data/pull/688 pull request to pure-data
- https://github.com/claudeha/libpd/tree/emscripten my fork of libpd with changes for Emscripten
- https://mathr.co.uk/empd/ my page about all these things

Thanks!

**Comment by Jonathhhan** [comment]
The Pure Data Emscripten branch was just merged. It would be very nice, if this will be compatible with ofxPd at some point.

**Comment by danomatika** [comment]
Current libpd master now uses pure-data master for testing which includes the now merged Emscripten support.

**Comment by danomatika** [comment]
@claudeha Is there any additional stuff needed for Emscripten, such as a different build target etc? Any specific info to add to the Readme about the platform?
